src/commands/force/org/open.ts

```diff
@@ -5,11 +5,10 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { EOL } from 'os';
-import { URL } from 'url';
 
 import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
-import { Messages, Org, MyDomainResolver, SfdxError, sfdc } from '@salesforce/core';
-import { Env, Duration } from '@salesforce/kit';
+import { Messages, Org, checkLightningDomain, SfdxError } from '@salesforce/core';
```

Comment:
Be aware that I forgot to add an export for `checkLightningDomain` here: https://github.com/forcedotcom/sfdx-core/blob/main/src/exported.ts and thus Im not sure you'll be able to load this dependency this way maybe if you do: `import checkLightningDomain from '@salesforce/core/util/checkLightningDomain'` maybe.

Reply:
we're keeping the checkLightningDomain for backward-compatibility reasons. This command can use the checkLightningDomain on sfdcUrl class (validates url, etc).

https://github.com/forcedotcom/sfdx-core/pull/420/files